test/test_jit.py

```diff
@@ -1222,7 +1222,7 @@ def forward(self, x):
         g = traced.graph
         torch._C._jit_pass_inline(g)
         torch._C._jit_pass_reconstruct_scopes(traced._c, g)
-        FileCheck().check("scope: top.sub.forward").run(g)
+        FileCheck().check("scope: top(MyModule).sub(SubModule).forward").run(g)
```

Comment:
Keep reconstruct scope related changes in base diff.

Reply:
That's what I intended to do, but things like `top(MyModule).sub(SubModule).forward` can't be tested on the base diff. Sorry about that.